plugins/kotlin/idea/src/org/jetbrains/kotlin/idea/intentions/RemoveAllArgumentNamesIntention.kt

```diff
@@ -0,0 +1,34 @@
+// Copyright 2000-2021 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+package org.jetbrains.kotlin.idea.intentions
+
+import com.intellij.openapi.editor.Editor
+import org.jetbrains.kotlin.idea.KotlinBundle
+import org.jetbrains.kotlin.idea.caches.resolve.resolveToCall
+import org.jetbrains.kotlin.idea.completion.placedOnItsOwnPositionInCall
+import org.jetbrains.kotlin.psi.KtCallElement
+import org.jetbrains.kotlin.psi.KtCollectionLiteralExpression
+import org.jetbrains.kotlin.psi.KtValueArgument
+import org.jetbrains.kotlin.resolve.calls.callUtil.getParameterForArgument
+import org.jetbrains.kotlin.resolve.calls.components.isVararg
+
+class RemoveAllArgumentNamesIntention: SelfTargetingIntention<KtCallElement>(
```

Comment:
I expect the intention to work for cases like this:
```
fun main() {
	foo(b = 1, a = 3)
}

fun foo(a: Int, b: Int) {
}
```
It should convert it to this:
```
fun main() {
	foo(3, 1)
}

fun foo(a: Int, b: Int) {
}
```